deploy-argocd-quicklab.sh

```diff
@@ -0,0 +1,54 @@
+API=upi-0.mptest.lab.upshift.rdu2.redhat.com:6443
+KUBEPASS=rSszg-hYQnn-h9zsi-aAoEb
+
+EMP="\e[1;4m"
+NORMAL="\e[0m"
+
+if which kustomize >/dev/null; then
+  kustomize=kustomize
+else
+  kustomize="toolbox run --container of-toolbox-v0.1.0 kustomize"
+fi
+
+if ! $kustomize --help > /dev/null; then
+  echo -e "${EMP}kustomize is not available --> aborting${NORMAL}"
+  exit 1
+fi
+
+# oc login -u kubeadmin -p rSszg-hYQnn-h9zsi-aAoEb upi-0.mptest.lab.upshift.rdu2.redhat.com:6443
+oc login -u kubeadmin -p $KUBEPASS $API
+
+echo -e "${EMP}Creating admin user 'myadmin'${NORMAL}"
+htpasswd -nb myadmin foobar68 > /tmp/oc.htpasswd
+oc create secret generic htpass-secret --from-file=htpasswd=/tmp/oc.htpasswd -n openshift-config
+cat <<EOF | oc apply -f -
+apiVersion: config.openshift.io/v1
+kind: OAuth
+metadata:
+  name: cluster
+spec:
+  identityProviders:
+  - name: my_htpasswd_provider
+    mappingMethod: claim
+    type: HTPasswd
+    htpasswd:
+      fileData:
+        name: htpass-secret
+EOF
+
+oc adm policy add-cluster-role-to-user cluster-admin myadmin
+
+oc login -u myadmin -p foobar68
+
+oc new-project argocd-test
+oc new-project aicoe-argocd-dev
+
```

Comment:
Same as crc, can you add gpg key import stuff.

Reply:
Right, I do not have the GPG step in the quicklab script. Adding it (the same it already exists in CRC) hope that's what you wanted.